src/PresentationalComponents/ExecutiveReport/BuildExecReport.js

```diff
@@ -147,7 +147,7 @@ const BuildExecReport = ({ asyncData }) => {
       {truncate(rule.summary, { length: 280 })}
     </Text>
   );
-
+  // TODO Replace insights with lightsped when figuring out how to.
```

Comment:
Yeah, I also thought about how to implement thi,s and I think it should be done not inthe  advisor but in the app that imports BuildExecReport module